Win32/Prj/wpcap.vcxproj

```diff
@@ -113,7 +113,8 @@
       <AdditionalDependencies>ws2_32.lib;..\..\..\..\packetWin7\Dll\Project\Release No NetMon and AirPcap\Packet.lib;%(AdditionalDependencies)</AdditionalDependencies>
     </Link>
     <PreBuildEvent>
-      <Command>win_flex -Ppcap_ -7 --outfile=..\..\scanner.c --header-file=..\..\scanner.h ..\..\scanner.l
+      <Command>call ..\..\..\GenVersion.bat
```

Comment:
What do you mean by "move all the files into libpcap"?  The generated `version.h` shouldn't be checked into the Git repository, and the generated `version.c` shouldn't be, either.

All you need to do is add `GenVersion.bat` as a file in the top-level directory, along with this change.  You can update this commit to include `GenVersion.bat`, as well as the changes to the project file, the way I described.